tests/features/project.feature

```diff
@@ -0,0 +1,43 @@
+@api @project
+Feature: Project content creation
+  In order to have projects on the site
+  As an editor
+  I need to be able to create and see project items
+
+  @javascript @av_portal
+  Scenario: Creation of a Project content through the UI.
+    Given I am logged in as a user with the "create oe_project content, access content, edit own oe_project content, view published skos concept entities, manage corporate content entities" permission
+
+    When I visit "the Project creation page"
+    And I fill in "Title" with "My Project"
+    And I fill in "Subject" with "EU financing"
+    And I fill in "Content owner" with "Committee on Agriculture and Rural Development"
+    And I fill in "Author" with "European Patent Office"
+    And I fill in "Departments" with "Audit Board of the European Communities"
+    And I fill in "Teaser" with "Project teaser"
+    And I fill in "Summary" with "Summary text"
+    And I fill in "Body text" with "Body text"
```

Comment:
please add check for "Funding Programme"

Reply:
Done